website/content/posts/introducing-action-validate-image.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "Introducing the latest addition Action Validate"
+date: 2023-10-23T13:02:00-04:00
+---
+
+Introducing [Action Validate](https://github.com/marketplace/actions/ec-validate).
+While you may already be familiar with using the [EC CLI](https://github.com/enterprise-contract/ec-cli) 
+and its `ec validate` command for local container image validation. Now, you can seamlessly integrate 
+this functionality directly into your build processes or any other automated workflows in GitHub.
+<!--more-->
+
+## What is EC Action Validate?
+
+**EC Action Validate** is a GitHub Action that brings validation capabilities of Enterprise Contract's EC CLI directly into your GitHub Workflow. Designed to integrate seamlessly into your build pipeline, this action offers both compliance assessments and intergrity checks for your container images. It ensures that your images meet both organizational and industry standards before deployment.
+
+### Key Features
+
+- **GitHub Native**: Being a GitHub Action, EC Action Validate seamlessly integrates into your existing GitHub workflows, while also provoiding github summary output.
+- **Policy Compliance**: ability to tailor its validation checks based on custom or pre-defined policies.
+- **Integrity Checks**: Verifies that the image hasn't been tampered with. 
+- **Authentication Support**: Offers support for both Long-Lived Public-Key Authentication and Keyless Authentication.
+  
+Interested in learning more? Visit our [EC Action Market Place](https://github.com/marketplace/actions/ec-validate) for a user guide.
+
+## The `Golden-Container` Build Pipeline Example using EC Action Validate 
+Imagine you're operating within a [build & release workflow](https://github.com/enterprise-contract/golden-container/blob/main/.github/workflows/release.yaml). You've successfully set up a pipeline that takes care of building, digitally signing, generating an SBOM, and adding provenance data. However, you're missing a crucial step validating the container image. This is where the `EC Action Validate` comes into play, ensuring your container images meet the required security and compliance standards before deployment.
+
```

Comment:
I think this needs a sentence that says you're going to walk-through the workflow before actually divining into it.

Reply:
```suggestion
Now, let’s go over the initial steps of this workflow before actually divining into it.
```

Reply:
"diving" not "divining" unless I'm missing something.. :crystal_ball: